semgrep/tests/e2e/rules/invalid-rules/invalid-metavariable-regex.yaml

```diff
@@ -9,4 +9,4 @@ rules:
         regex: (boto|boto3)
   message: "Boto3 connection to internal network"
   languages: [python]
-  severity: ERROR
\ No newline at end of file
+  severity: ERROR
```

Comment:
These missing newlines caused the `invalid-rules` snapshots below to change. I'm not sure how important it is that these stay the same. Note that the snapshot changes are under the top-level `errors` key, not `results`, so this shouldn't affect downstream applications.